### ISSUE TITLE
"Feature" Suggestion (Low Priority, Developer QoL Question rather than WCAG related)

### ISSUE THREAD

**Issue by rainbowforgedev** [accessiblepixel/oneclickaccessibility2-fork#3]
Hello again! This is not so much an issue as a question about a potential change to the customizer options.

I really wanted to use variables for the toolbar colors etc. rather than the default color picker, and now that I've got a plugin that my clients won't be able to update, I wanted to test out making some changes to it :) so I changed all the "color" field values in https://github.com/accessiblepixel/oneclickaccessibility2-fork/blob/main/includes/pojo-a11y-customizer.php to text fields instead so I could just use variables, and it seems to render the css just fine. I thought I'd put this comment here for the following reasons:

1. In case you or someone else might be interested in using it the same way (grr color picker without custom palette), Or perhaps
2. In case there's some other way to achieve similar functionality (for eg. I'm not sure how difficult it is to create a customizer object that has the theme's custom color palette available within plugin settings?) If it's a case of I just haven't read https://developer.wordpress.org/themes/customize-api/customizer-objects/ enough, and it's basically just a different premade object, it'd probably make more sense to just do that? If text is the easier way, then perhaps
3. In case there's a security reason that I Shouldn't do what I've done, maybe you or someone else will be kind enough to let me know that :>

Thank you so very very much for doing this once again!!!

P.S. I unfortunately don't have funds to spare at the moment 😢  but if you put a donation link of some kind in the customizer, I bet people who are used to Elementor shoving itself everywhere will be just fine with that.

**Comment by accessiblepixel** [comment]
@rainbowforgedev 
Had a bit more of a thought about this, and it should be possible to have a "Use colour picker" "don't use colour picker" in the customiser, would probably just require a new variable and option choice within the setup section to choose which mode.

And then have the customiser check that preference before hand.

Okay. I've implemented something in the v2.2.7-beta branch - there's a new option in the 'settings' page to choose if the colour picker is displayed within the customiser section, or if it's treated like a text field.

Can  you let me know if this satisfies your FR and... works? :)

Thanks.

**Comment by rainbowforgedev** [comment]
Hi!!
My apologies for the delayed response! I think this sounds like an incredible update and I can't wait to try it out!! Thank you so much again for all of your hard work maintaining this plugin <3 I really hope that more people find out about it and start to use it!

Edit: Never mind lol I just saw your other thread where you mentioned being flagged as a spammer by WP code review team :/ this kinda issue is why I'm primarily moving to Svelte for as much of my work as I can :) so I'm eager to try out whether I can use your static site idea too :>

**Comment by accessiblepixel** [comment]
No problem. :) I hope you're well.

I've tested v2.2.7 on my sites since I committed the changes, and haven't encountered any problems with the new options, so I pushed it as a release today when I saw your message.

RE: WP.org - I've thought about it, but last time I just ended up infuriated. My account got flagged by an automated system as spam, then it went through human review and they also said I was spam.... so... I'll maybe try again at some point but... rawrrrrr :<

**Comment by accessiblepixel** [comment]
Rather than editing my previous, because I didn't see your edit haha

Oh, yes, the static site idea. I've got it mostly working on a faircamp site I run (think static site like bandcamp) and that seems to be going well, so I'm getting closer to having at least... the knowledge on how to make it work independently, how best to share that or implement that in a generic way... Not yet 😆
